CHANGELOG.md

```diff
@@ -434,6 +434,12 @@ Released with 1.0.0-beta.37 code base.
 
 ## [Unreleased]
 
+## [1.5.3]
+
+### Fixed
+
+- Unable to send legacy transaction if network supported EIP-1559 (#4277w)
```

Comment:
Plus add more description of current behavior for release notes and doc: 

"If `tx.type` is not specified web3 will consider it type `0x2` transaction ( provided NW supports this ) and will add  `maxFeePerGas`, `maxPriorityFeePerGas`. "

other thn that lgtm